Automattic/wp-calypso
# Switch the dialog backdrop to dark in blazepress modal

#### Proposed Changes

Resolves [this issue](https://github.tumblr.net/Tumblr/wordads-picard/issues/609)

Before version 
![Screenshot 2022-10-31 at 13 41 26](https://user-images.githubusercontent.com/1416426/198999933-655ceb81-d845-4ecc-aedb-fa7e38937967.png)


after version 
![Screenshot 2022-10-31 at 13 41 03](https://user-images.githubusercontent.com/1416426/198999959-afd31320-d0fe-4db6-92cb-a589afc3f3f6.png)


#### Testing Instructions

Checkout branch and spin it up in your localhost.. open up a promote campaign dialog and check the backdrop styles 

#### Pre-merge Checklist






Related to #
